lib/beaker-pe/pe-client-tools/executable_helper.rb

```diff
@@ -75,7 +75,9 @@ def login_with_puppet_access_on(host, credentialed_dispatcher, opts={})
 
             user = credentialed_dispatcher.credentials.login
             password = credentialed_dispatcher.credentials.password
-            puppet_access_on(host, 'login', {:stdin => "#{user}\n#{password}\n"})
+            args = ['login']
+            args.push "--lifetime #{lifetime}" if lifetime
```

Comment:
This should be fine.  Beaker command contains this method for arguments:
    # @param [Array] args An array of arguments to the command.
    #
    # @return [String] String of the arguments for command.
    def args_string args = @args
      args.flatten.compact.join(' ')
    end

So a single string containing separated by a space, or two strings in an array would have the same end result.